pypeit/spectrographs/vlt_fors.py

```diff
@@ -121,20 +121,23 @@ def compound_meta(self, headarr, meta_key):
             binning = parse.binning2string(binspec, binspatial)
             return binning
         elif meta_key == 'decker':
-            mode = headarr[0]['HIERARCH ESO INS MODE']
-            if mode in ['LSS', 'MOS']:
-                try:  # Science
-                    return headarr[0]['HIERARCH ESO INS SLIT NAME']
-                except KeyError:  # Standard!
-                    try:
-                        return headarr[0]['HIERARCH ESO SEQ SPEC TARG']
-                    except KeyError:
-                        return headarr[0]['HIERARCH ESO INS MOS CHECKSUM']
-            elif mode == 'IMG':
-                # This is for the bias frames
-                return None
-            else:
-                msgs.error(f"PypeIt does not currently support VLT/FORS2 '{mode}' data reduction.")
+            try:
+                return headarr[0]['DECKER']
```

Comment:
Sorry to continue to be nit-picky, but I can't help myself.  Personally, I'd rather we use if statements instead of these try/except blocks.  To keep the change simple, could we replace the two lines above with:
```
if 'DECKER' in headarr[0]:
    return headarr[0]['DECKER']
```
Then we can leave the rest of the code as it was before.

Reply:
That's what I get for trying to maintain a consistent style ;) no problem!